dashboards/traefik/traefik.json

```diff
@@ -249,7 +249,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nlatest(traefik_service_open_connections) AS 'Open Connections',\nsum(traefik_service_requests_total) AS 'Total Requests',\nsum(traefik_service_requests_tls_total) AS 'Total TLS Requests',\nbucketPercentile(traefik_service_request_duration_seconds_bucket, 95) AS 'Request Duration (ms)',\naverage(traefik_service_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET prometheus_server, service, method, protocol\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\nlatest(traefik_service_open_connections) AS 'Open Connections',\nsum(traefik_service_requests_total) AS 'Total Requests',\nsum(traefik_service_requests_tls_total) AS 'Total TLS Requests',\nbucketPercentile(traefik_service_request_duration_seconds_bucket, 95) AS 'Request Duration (ms)',\naverage(traefik_service_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET prometheus_server, service, method, protocol\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance)"
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', service, method, protocol
```

Reply:
Thanks, changes made.